src/_ekeyword

```diff
@@ -0,0 +1,36 @@
+#compdef ekeyword
+
+local -a arguments=(
+    '(- :)'{-h,--help}'[Show this help message and exit]'
+    '(-m --manifest)'{-m,--manifest}'[Run `ebuild manifest` on the ebuild after modifying it]'
+    '(-n --dry-run)'{-n,--dry-run}'[Show what would be changed, but do not commit]'
+    '(-v --verbose)'{-v,--verbose}'[Be verbose while processing things]'
+    '(-q --quiet)'{-q,--quiet}'[Be quiet while processing things (only show errors)]'
+    '(--format)'--format':Select output format for showing differences:_values -V ekeywordsfmt "ekeywords formats" auto color-inline inline short-multi long-multi'
+    '(- :)'{-V,--version}'[Show version information]'
+)
+
+(( $+functions[_ekeywordargs] )) ||  _ekeywordargs() {
+    _files -g \*.ebuild
+    (( $words[(I)(--format)] )) && return
+
+    # ${PORTDIR}/profiles/arch.list
+    local -a keywords=(
```

Comment:
This should use the arch list provided by _gentoo_arches which pulls directly from the repo's arch.list instead of hardcoding the list again here.

Reply:
Yes, first version has been with arch.list.
however didnot found a good / fast way to get PORTDIR -  `portageq get_repo_path / gentoo` is quite slow.

Reply:
initial this PR focused on further improvements of the generic `eselect` completion - what do you think / any comments on that?